### ISSUE TITLE
Print Text Block instead of Print Lines

### ISSUE THREAD

**Issue by jsc-dsacharow** [karolyi/py3-ecrterm#8]
Hi, 

Thank you so much for your work!
I'm currently looking into your project and trying to improve the speed when the ECR is printing the receipts. 
Unfortunately, the Print Lines method is quite slow which brought me to "Print Text Block" but I can't seem to make it work.

Could you point me in the right direction where to start first?
I understand that I need to use TLV Containers but there's just no comparable function where I could get guidance.

I'd really appreciate your input here.

Thank you very much!  

EDIT: To further clarify: I've already read through the ZVT-Protocol Document. It says I have to "allow" the PT to use this packet, but that's as far as I got. Do I have to allow the bitmap "tlv" in the Authorization packet?

**Comment by karolyi** [comment]
Hey,

I haven't touched this codebase for a long time, as the product I needed this to build for is now functioning. There was a guy who started implementing the TLV handling, and I took the chance to merge his changes into a branch that can be looked at by you if you like: https://github.com/karolyi/py3-ecrterm/tree/py3

Please look into this, you might find some clues to your questions. If have something working with it, please feel free to contact me again so I can merge your changes back so we can extend the functionality of this module further.

**Comment by jsc-dsacharow** [comment]
Hi and thanks for your quick reply.

I'll take a look at the branch. 
.. could take a bit longer since I've fried (Boot-Loop) my PT by restarting it from my ECR using the Reset Packet.
Ingenico and I are not sure how it could happen since the Packet was right.

TL;DR: I'll take a look and try to contribute once I've managed to implement it. 
Thanks!

**Comment by karolyi** [comment]
Hey,

I just merged @henryk's changes back into py3 so it might be even more advanced than it was before. I think it's worth looking into it, and he might be of help as well.

As for your terminal, sorry to hear that. As a general saying, software is a mess. Probably there must have been something ingenico didn't pay attention to when building the firmware.

Let me know whenever you have something to contribute back here, and good luck with your endeavor!
